Integrations/MailSenderNew/MailSenderNew.yml

```diff
@@ -101,4 +101,5 @@ script:
     description: Send an email
   runonce: false
 tests:
-  - Mail Sender (New) Test
\ No newline at end of file
+  - Mail Sender (New) Test
+releaseNotes: "Support for Internationalized Headers. Fix for new lines in message subject."
```

Comment:
release notes should be in a separate file now

Reply:
(pull from master)

Reply:
Done